netlify.toml

```diff
@@ -1,6 +1,6 @@
 [build]
   # This will be run the site build
-  command = "cd site && npm install && npm run build"
+  command = "npm install && cd site && npm install && npm run build"
```

Comment:
Does the build bot not install top level automatically when you cd around and npm i?

Reply:
TIL